src/components/load/LoadByRDFForm.jsx

```diff
@@ -40,19 +40,31 @@ const LoadByRDFForm = (props) => {
     event.preventDefault()
   }
 
+  const n3PlaceHolder = `For example:
+<> <http://id.loc.gov/ontologies/bibframe/mainTitle> "Tractatus Logico-Philosophicus"@en .
+<> <http://sinopia.io/vocabulary/hasResourceTemplate> "resourceTemplate:bf2:WorkTitle" .
+<> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://id.loc.gov/ontologies/bibframe/Title> .
+  `
+
+  const baseURIPlaceholder = 'For example: https://trellis.sinopia.io/repository/stanford/e111a712-5a45-4c2a-9201-289b98d7452e.'
+
   return (
     <div>
-      <h4>Load RDF</h4>
+      <h4>Load RDF into Editor</h4>
       <form id="loadForm" onSubmit={handleSubmit}>
         <div className="form-group">
           <label htmlFor="resourceTextArea">N3 RDF</label>
-          <textarea className="form-control" id="resourceTextArea" rows="15" value={resourceN3} onChange={handleResourceN3Change}></textarea>
+          <textarea className="form-control" id="resourceTextArea" rows="15" value={resourceN3}
+                    onChange={handleResourceN3Change} placeholder={n3PlaceHolder}></textarea>
         </div>
         <div className="form-group">
           <label htmlFor="uriInput">Base URI</label>
-          <input type="text" className="form-control" id="uriInput" value={baseUri} onChange={handleBaseUriChange} />
+          <input type="text" className="form-control" id="uriInput" value={baseUri} onChange={handleBaseUriChange}
```

Comment:
`type` should be `url` See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url

Reply:
This is why I love code review. Always learning something new. (Or reminded of something I once knew.)

Changed.